Ktisis/Interface/Windows/References.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+using Dalamud.Logging;
+
+using ImGuiNET;
+using ImGuiScene;
+
+namespace Ktisis.Interface.Windows {
+	internal static class References {
+		public static Dictionary<int, TextureWrap> Textures = new();
```

Comment:
See other comment -- though here I have an alternate suggestion: turn this into a `string`-indexed `Dictionary` and keep a ref-count for how many `ReferenceInfo`s use a given texture. 
Dispose when deleting the reference that uses the texture.

Basically just deduplication if someone loads the same image as two references. Probably a bit overkill, but might still be nice to have.

Reply:
I've opted for `DisposeUnreferencedTextures` instead of a formal refcount.

Reply:
That also works. (though it's technically less performant, for the expected data size that *should* be okay). Resolving.